app/src/main/java/com/example/moviesapp/util/Constants.kt

```diff
@@ -1,11 +1,13 @@
-package com.example.moviesapp.utils
+package com.example.moviesapp.util
 
 object Constants {
     const val TAB_NOW_PLAYING = "NOW PLAYING"
     const val TAB_TOP_RATED = "TOP RATED"
     const val TAB_UPCOMING = "UPCOMING"
-    const val EMPTY_DATABASE = "EMPTY_DATABASE"
+    const val IMAGE_RADIUS = 20
     const val FIRST_POSITION = 0
     const val SECOND_POSITION = 1
     const val NUMBER_OF_PAGES = 3
+    const val ERROR = "Something got wrong, please try again later"
```

Comment:
if you will show this method on the screen, this should be in the string res, nor here

Reply:
done